plugins/inputs/opcua/README.md

```diff
@@ -85,39 +105,61 @@ Plugin minimum tested version: 1.16
   ## namespace, this is used.
   # identifier_type =
   #
+  ## Default tags that are applied to every node in this group. Can be
+  ## overwritten in a node by setting a different value for the tag name.
+  ##   example: default_tags = { tag1 = "value1" }
+  # default_tags = {}
+  #
   ## Node ID Configuration.  Array of nodes with the same settings as above.
+  ## Use either the inline notation or the bracketed notation, not both.
+  #
+  ## Inline notation (default_tags not supported yet)
   # nodes = [
-  #  {name="", namespace="", identifier_type="", identifier=""},
-  #  {name="", namespace="", identifier_type="", identifier=""},
+  #  {name="node1", namespace="", identifier_type="", identifier=""},
+  #  {name="node2", namespace="", identifier_type="", identifier=""},
   #]
+  #
+  ## Bracketed notation
+  # [[inputs.opcua.group.nodes]]
+  #   name = "node1"
+  #   namespace = ""
+  #   identifier_type = ""
+  #   identifier = ""
+  #   default_tags = { tag1 = "override1", tag2 = "value2" }
+  #
+  # [[inputs.opcua.group.nodes]]
+  #   name = "node2"
+  #   namespace = ""
+  #   identifier_type = ""
+  #   identifier = ""
 
   ## Enable workarounds required by some devices to work correctly
   # [inputs.opcua.workarounds]
     ## Set additional valid status codes, StatusOK (0x0) is always considered valid
     # additional_valid_status_codes = ["0xC0"]
 
+  # [inputs.opcua.request_workarounds]
     ## Use unregistered reads instead of registered reads
     # use_unregistered_reads = false
 ```
```

Comment:
Moving use_unregistered_reads from the OpcuaWorkarounds struct to the new ReadClientWorkarounds struct makes sense from the perspective of factoring ReadClient into common, but it's not a backwards compatible change. Existing users of use_unregistered_reads who upgrade won't be able to use their old configuration file without modification.

The changes to inputs.opcua need to be backward compatible for this to be merged and released in the next feature release.

Reply:
@reimda I wondered this also, but it appeared it is an unreleased feature yet. So as long this PR gets merged before next feature release we are fine.

Reply:
Oh ok, I missed that. Thanks for pointing it out! I guess I didn't see it because the review comments were collapsed as resolved.